src/bytes/complete.rs

```diff
@@ -46,6 +82,25 @@ where
   }
 }
 
+/// Parse till certain characters are met
+///
+/// The parser will return the longest slice till one of the characters of the combinator's argument are met.
+///
+/// It doesn't consume the matched character,
+///
+/// It will return a `Err::Incomplete(Needed::Size(1))` if the pattern wasn't met
```

Comment:
since we're in the complete version, it should return an error, not `Incomplete`